fontbe/src/kern.rs

```diff
@@ -64,8 +74,25 @@ impl Work<Context, AnyWorkId, Error> for KerningWork {
 
         let mut builder = PairPosBuilder::default();
 
+        // Add IR kerns to builder. IR kerns are split by location so put them back together again.
+        // We want to iterate over (left, right), map<location: adjustment>
+        let mut kerns: HashMap<&KernPair, Vec<(NormalizedLocation, OrderedFloat<f32>)>> =
```

Comment:
I'm also typing here right now, and something I'm wondering is whether it makes sense to group everything by glyph in the IR? my particular observation is that building this data structure is slow, and we could build the full data structure up after we do alignment.

Reply:
https://github.com/googlefonts/fontc/pull/644 suggests once sharded kerning isn't too bad, lets keep that in our back pocket until it pops out in a profile. Does the fea compilation contain any opportunity for ||ism? - it ends up being quite long, as does packing tables (https://github.com/googlefonts/fontc/issues/610).

Reply:
I suggest we resolve here and pursue in #610